keps/sig-node/2133-kubelet-credential-providers/kep.yaml

```diff
@@ -27,12 +28,13 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: "v1.25"
```

Comment:
please also update the https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-node/2133.yaml

Reply:
Done, PTAL